receptor.c

```diff
@@ -55,7 +55,7 @@ bindlisten(
 	struct addrinfo *res, *resw;
 	char buf[128];
 	char saddr[INET6_ADDRSTRLEN];
-	int err;
+	int err, binderr = 0;
```

Comment:
Nitpicking, but could you put this one on a line by itself?  Also I usually use a char for these.